core/src/main/scala/org/ensime/indexer/orientdb/SimpleOrient.scala

```diff
@@ -372,7 +373,7 @@ package object syntax {
       bdf: BigDataFormat[FqnSymbol],
       oid: OrientIdFormat[FqnSymbol, P],
       p: SPrimitive[P]
-    ): Seq[VertexT[FqnSymbol]] = {
+    ): Seq[(VertexT[FqnSymbol], VertexT[LineNumber])] = {
```

Comment:
this also seems to be doing two jumps in one go.

It looks like the generalisability of this file went out the window a long time ago, it was never supposed to know anything about the domain object but it seems to be hardcoding `FqnSymbol` and `UsedIn`.